tests/test_core.py

```diff
@@ -6127,7 +6127,6 @@ def test_bullet(self, use_cmake):
     if WINDOWS and not use_cmake:
       self.skipTest("Windows cannot run configure sh scripts")
 
-    self.set_setting('DEAD_FUNCTIONS', ['__ZSt9terminatev'])
```

Comment:
Any idea what this was doing here?

Reply:
I'm pretty sure a long time ago our libc was missing that, so this avoided a link error on fastcomp. But I think it's been unnecessary for a long time...